wordpress-mobile/WordPress-iOS
# Testing: Zip and save xcresult as artifact

This updates our CircleCI config so that on failed unit and UI test runs the .xcresult file is saved as an artifact, so we can download it and view the test report in Xcode.

To test:

* Check the artifacts for a failed test run (e.g. a [unit test run](https://circleci.com/gh/wordpress-mobile/WordPress-iOS/16124) or [UI test run](https://circleci.com/gh/wordpress-mobile/WordPress-iOS/16125)) and ensure you can find an `xcresult.zip` file in the artifacts.
* Open the zip file, ensure there is a `test-without-building.xcresult` file there, and confirm you can open it in Xcode to view the test report from the CI test run.

Update release notes:
